Bikes/Bikes.Domain/Enum/BikeType.cs

```diff
@@ -0,0 +1,37 @@
+﻿namespace Bikes.Domain.Enum;
+
+/// <summary>
+/// Type of bike
+/// </summary>
+public enum BikeType
+{
+    /// <summary>
+    /// Road bike
+    /// </summary>
+    Road,
+
+    /// <summary>
+    /// Mountain bike
+    /// </summary>
+    Mountain,
+
+    /// <summary>
+    /// Hybrid bike
+    /// </summary>
+    Hybrid,
+
+    /// <summary>
+    /// Sport bike
+    /// </summary>
+    Sport,
+
+    /// <summary>
+    /// Electric bike
+    /// </summary>
+    Electric,
+
+    /// <summary>
+    /// BMX bike
+    /// </summary>
+    BMX
```

Comment:
Акронимы также необходимо именовать в PascalCase.

```suggestion
    Bmx
```